src/common/components/dashboard/components/dashboardItem.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import {Link} from 'react-router';
+const styles: any = require('./dashboardItemStyles.scss');
```

Comment:
What do you think if I rename `styles` to `classNames`? 
`styles` could be confused by `style` property

@brauliodiez